.circleci/linux_circle_vm_setup.sh

```diff
@@ -33,4 +33,8 @@ if ! docker buildx inspect ddev-builder-multi --bootstrap >/dev/null; then docke
 docker buildx inspect --bootstrap
 
 # Install github's gh tool
-wget -O /tmp/gh.deb https://github.com/cli/cli/releases/download/v2.1.0/gh_2.1.0_linux_${ARCH}.deb && sudo dpkg -i /tmp/gh.deb >/dev/null
+curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+&& sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+&& echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+&& sudo apt -qq update \
+&& sudo apt -qq install gh -y
```

Comment:
I definitely didn't understand why you were installing gh here.

Reply:
I changed the installation method to apt only because I always prefer apt over other installation methods.

Reply:
The question was why you were installing `gh`, why is it needed?